### PR TITLE
Jbuilder

### PR DESCRIPTION
**Pull request by dinosaure**
Move to `jbuilder`, fixed CI and integrate patch of @wictory.

### CODE REVIEW

**Comment by dinosaure** [comment]
Currently, `Syndic` works on `ocaml.4.01.0` and upper. However, `ocaml.4.03.0` integrate a breaking change about `String.lowercase` and `String.lowercase_ascii`. I decided to move (and constraint) `Syndic` to be available only on `>= 4.03.0` but I'm not sure about this choice.

@Chris00 it's fine for you? Keep the compatibility with `4.01.0` is not a problem for me.

**Comment by dinosaure** [comment]
Note that `jbuilder` is available only on `>= 4.02.3`.

**Comment by Chris00** [comment]
> @Chris00 it's fine for you?

Fine with me.

File: syndic.opam [hunk 1]
```diff
@@ -7,20 +7,25 @@ license:      "MIT"
 homepage:     "https://github.com/Cumulus/Syndic"
 dev-repo:     "git+https://github.com/Cumulus/Syndic.git"
 bug-reports:  "https://github.com/Cumulus/Syndic/issues"
+synopsis:     "RSS1, RSS2, Atom and OPML1 parsing"
+description: """Pure OCaml Library for parsing and writing various types of
```
Comment (Chris00) [inline syndic.opam]: I think it is nice to have a blank line between the synopsis and the description.  Hence:
```
description: """
Pure OCaml...
```

**Comment by Chris00** [comment]
This is no longer Jbuilder but Dune! ;-)

**Comment by dinosaure** [comment]
samarsheoam ©

**Pull request merged**